ports/esp32/machine_pin.c

```diff
@@ -43,7 +43,7 @@
 #include "modesp32.h"
 #include "genhdr/pins.h"
 
-#if CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32C6
+#if SOC_USB_SERIAL_JTAG_SUPPORTED
```

Comment:
Looks like ESP32S3 also defines `SOC_USB_SERIAL_JTAG_SUPPORTED`, but I guess that's OK, it'll just include the header and won't change any functionality.

Reply:
Some people build S3 with USB Serial/JTAG peripheral instead of the native one, as well. (No supported boards do this, but we've had bug reports for it.)